sirepo/auth/__init__.py

```diff
@@ -146,32 +152,18 @@ def get_module(name):
 
 
 def init_apis(*args, **kwargs):
-    global uri_router, simulation_db, visible_methods, valid_methods, non_guest_methods
-    assert not _METHOD_MODULES
-
+    global uri_router
+    assert not uri_router
```

Comment:
init_apis is guaranteed to not be called twice so no need for an assertion even though it seems like a good idea. The reason is that a contract is a contract and if init_apis were called twice anywhere, other things would break so that's a contract we can trust, not assert, and advertise that by not asserting.

Reply:
c1482e38